src/app/webcontainer/webapp-container.qml

```diff
@@ -192,7 +192,12 @@ BrowserWindow {
             DomainPermissionsModel.whiteListMode = settings.domainWhiteListMode;
             DomainSettingsModel.databasePath = webappDataLocation + '/domainsettings.sqlite';
             DomainSettingsModel.defaultZoomFactor = settings.zoomFactor;
+            DownloadsModel.databasePath = webappDataLocation + "/downloads.sqlite";
             UserAgentsModel.databasePath = DomainSettingsModel.databasePath;
+
+            // this can be set from QtWebEngine version 1.9 (Qt 5.13)
+            //item.currentWebview.profile.downloadPath = webappDataLocation + "/Downloads";
+            FileOperations.mkpath(webappDataLocation + "/Downloads");
```

Comment:
Could you also add the note that the part in WebApp.qml can be deleted when this is enabled? Maybe file an issue to fix both and reference that issue here.